examples/flax/image-captioning/run_image_captioning_flax.py

```diff
@@ -182,15 +183,24 @@ class ModelArguments:
             )
         },
     )
-    use_auth_token: bool = field(
-        default=False,
+    token: str = field(
+        default=None,
         metadata={
             "help": (
                 "Will use the token generated when running `huggingface-cli login` (necessary to use this script "
                 "with private models)."
```

Comment:
The help is not accurate anymore here. Will use the `token` if it is passed, and the token generated when running `huggingface-cli login` if not passed. The parenthesis is not applicable anymore since we always look at the token generated when running `huggingface-cli login` if `token` is left to `None`.